src/platform_impl/windows/event.rs

```diff
@@ -388,8 +388,8 @@ pub fn process_key_params(
     wparam: WPARAM,
     lparam: LPARAM,
 ) -> Option<(ScanCode, Option<VirtualKeyCode>)> {
-    let scancode = ((lparam >> 16) & 0xff) as UINT;
     let extended = (lparam & 0x01000000) != 0;
+    let scancode = if extended { 0xE000 } else { 0x0000 } | ((lparam >> 16) & 0xff) as UINT;
     handle_extended_keys(wparam as _, scancode, extended)
```

Comment:
Seems like the method `handle_extended_keys` exists specifically to handle extended keys. So it might be appropriate to move this inside of it?

Reply:
:+1: I've moved it in to `handle_extended_keys`